lib/appium.js

```diff
@@ -344,32 +346,136 @@ class AppiumDriver extends BaseDriver {
     }
   }
 
+  pluginsToWrapCmd (cmd) {
+    return this.plugins.filter((plugin) => plugin.wrapCommands.includes(cmd));
+  }
+
+  pluginsToHandleCmd (cmd) {
+    return this.plugins.filter((plugin) => {
+      if (plugin.commands === true) {
+        return true;
+      }
+      if (_.isArray(plugin.commands) && plugin.commands.includes(cmd)) {
```

Comment:
`_.includes(plugin.commands, cmd)` would do the job

Reply:
the whole method could be rewritten to

`return this.plugins.filter((p) -> plugin.commands === true || _.includes(plugin.commands, cmd))`